### PR TITLE
fix: remove select text from multiselect application questions

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

This PR addresses #3009

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Remove redundant select language from above application multiselect questions.

## How Can This Be Tested/Reviewed?

View multiselect questions (household, programs, preferences) in an application and ensure the text is no longer present.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/pages/applications/household/changes.tsx [hunk 1]
```diff
@@ -102,11 +102,10 @@ const ApplicationHouseholdChanges = () => {
         <Form onSubmit={handleSubmit(onSubmit, onError)}>
           <div
             className={`form-card__group field text-lg ${
-              errors.expectingHouseholdChanges ? "error" : ""
+              errors.householdExpectingChanges ? "error" : ""
```
Comment (emilyjablonski) [inline sites/public/pages/applications/household/changes.tsx:105]: Unrelated bug, we were never showing errors on this page

**Review by ludtkemorgan — APPROVED** [review]
👍

**Review by ColinBuyck — CHANGES_REQUESTED** [review]
Looks like there's still one in the ada.tsx file. Other than that, everything looks good!
Screen Shot 2022-08-30 at 3 20 08 PM

**Comment by emilyjablonski** [comment]
@ColinBuyck Good catch thank you! ⭐

**Review by seanmalbert — APPROVED** [review]

**Review by ColinBuyck — APPROVED** [review]
LGTM!

**Pull request merged**